app_allocator/classes/read_need.py

```diff
@@ -0,0 +1,25 @@
+class ReadNeed(object):
+    def __init__(self, count):
+        self.count = count
+
+    def __str__(self):
+        return "R(%s)" % self.count
```

Comment:
I don't really love this convention. Could we maybe use words?

Reply:
Again this was really a debugging thing so I could comprehend the output.

Reply:
not a big deal - if it's more convenient this way that's fine